src/scss/base/_layout.scss

```diff
@@ -1,3 +1,8 @@
+@use '@justeat/pie-design-tokens/dist/jet'; as dt;
```

Comment:
Just wondered what the reasoning was for namespacing these un `dt` rather than importing the raw variables with `*`?

Reply:
Yep – the main reason is that it makes it super clear what variables are local to the file and what variables are being imported in from another module. So here, anything namespaced as `dt`, is a design token variable.

We talked about whether we should just use `*` for imported variables, but overall I think having the distinction makes it clearer to new people coming to the files.

For consuming applications, it'll be their own choice whether they want to namespace the variables passed through by fozzie with `@forward` or not when they `@use` fozzie.

Reply:
Thanks, @ashleynolan, makes sense to me.